ogr/ogrsf_frmts/aivector/ograivectordriver.cpp

```diff
@@ -104,14 +104,14 @@ static GDALDataset *OGRAIVectorOpen(GDALOpenInfo *poOpenInfo)
       public:
         MyDataset() = default;
 
-        int GetLayerCount() override
+        int GetLayerCount() const override
         {
             return 1;
         }
 
-        OGRLayer *GetLayer(int idx) override
+        OGRLayer *GetLayer(int idx) const override
```

Comment:
Should the interface have both

`const OGRLayer *GetLayer(int idx) const` and `OGRLayer *GetLayer(int idx)` 

instead of 

`OGRLayer *GetLayer(int idx) const` ?

Reply:
done with latest commit